controller/volume_controller.go

```diff
@@ -430,6 +432,15 @@ func (vc *VolumeController) syncVolume(key string) (err error) {
 	return nil
 }
 
+// handleConditionLastTransitionTime rollback to the existing condition object if condition's value hasn't changed
+func handleConditionLastTransitionTime(existingStatus, newStatus *types.VolumeStatus) {
+	for conditionType, condition := range newStatus.Conditions {
+		if existingCondition, ok := existingStatus.Conditions[conditionType]; ok && existingCondition.Status == condition.Status {
```

Comment:
Also need to check the reason and message to ensure they're the same one.

Reply:
Thank you! Fixed